client/tests/integration/triggers/time_trigger.rs

```diff
@@ -34,6 +34,9 @@ fn time_trigger_execution_count_error_should_be_less_than_15_percent() -> Result
     wait_for_genesis_committed(&vec![test_client.clone()], 0);
     let start_time = current_time();
 
+    // Start listening BEFORE submitting any transaction not to miss any block committed event
+    let event_listener = get_block_committed_event_listener(&test_client)?;
```

Comment:
So this test is ignored, but you fixed it too? Interesting. Mabye we should unignore it if it works fine now?

Reply:
The whole point is to unignore these tests. We only remove the tests that are redundant after the Pytest 
sdk was merged